pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu

```diff
@@ -40,6 +40,7 @@ ENV DGLBACKEND=pytorch
 ENV CMAKE_PREFIX_PATH="$(dirname $(which conda))/../"
 ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
 ENV MANUAL_BUILD=0
+ENV RDMAV_FORK_SAFE=1
```

Comment:
Did replacing EFA's installation of OMPI with built from source still having the fork() issue?

Reply:
`RDMAV_FORK_SAFE=1` is still needed for programs using RDMA outside MPI (NCCL and Herring for example). That flag won’t affect MPI behavior anymore since MPI knows nothing about EFA now

Reply:
Make sense, thanks.